doc/guide/r_quickstart.hpp

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file r_quickstart.hpp
+ * @author Yashwant Singh Parihar
+
+@page r_quickstart mlpack in R quickstart guide
+
+@section r_quickstart_intro Introduction
+
+This page describes how you can quickly get started using mlpack from R and
+gives a few examples of usage, and pointers to deeper documentation.
+
+This quickstart guide is also available for @ref python_quickstart "Python"
+@ref cli_quickstart "the command-line", @ref julia_quickstart "Julia" and
+@ref go_quickstart "Go".
+
+@section r_quickstart_install Installing mlpack
+
+Installing the mlpack bindings for R is straightforward; you can just use
+cran mirror:
+
+@code{.R}
+install.packages('mlpack')
+@endcode
+
+Building the R bindings from scratch is a little more in-depth, though.  For
+information on that, follow the instructions on the @ref build page, and be sure
+to specify @c -DBUILD_R_BINDINGS=ON to CMake; you may need to also set the
+location of the R program with @c -DR_EXECUTABLE=/path/to/R.
+
+@section r_quickstart_example Simple mlpack quickstart example
+
+As a really simple example of how to use mlpack from R, let's do some
+simple classification on a subset of the standard machine learning @c covertype
+dataset.  We'll first split the dataset into a training set and a testing set,
+then we'll train an mlpack random forest on the training data, and finally we'll
+print the accuracy of the random forest on the test dataset.
+
+You can copy-paste this code directly into R to run it.
+
+@code{.R}
```

Comment:
As `data.table` isn't part of base, we'll need to install it.

```suggestion
@code{.R}
if(!requireNamespace("data.table", quietly = TRUE)) { install.packages("data.table") }
```

Reply:
Thank you :)